WordPress/src/main/res/values/strings.xml

```diff
@@ -540,7 +540,7 @@
     <string name="site_settings_privacy_title" translatable="false">@string/privacy</string>
     <string name="site_settings_language_title" translatable="false">@string/language</string>
     <string name="site_settings_username_title" translatable="false">@string/username</string>
-    <string name="site_settings_password_title" translatable="false">@string/password</string>
+    <string name="site_settings_password_title">Update password</string>
```

Comment:
Changing the `site_settings_password_title` resource value changes not only the dialog, but also the preference title.  Was that intentional?  That makes it a little inconsistent with the ***Username*** preference.  Since long-pressing those preferences provides a hint for the action, should we leave the preferences ***Username*** and ***Password***?  @michelleweber, to keep the fields consistent, would it make sense to have just ***Password*** as the dialog title since there is a description of what the action does in the dialog message?

Reply:
I like the added clarity of having the added word in the title, since this is a very particular kind of password action, but if it will eff with other things to add it, leave it off. It's not that big a whoop.